.github/workflows/build-nightly.yml

```diff
@@ -41,3 +42,25 @@ jobs:
         type=raw,value=nightly
         type=schedule,pattern=${{ needs.compute-suffix.outputs.prerelease }}
     secrets: inherit
+
+  build-image-streaming:
+    needs: compute-suffix
+    uses: ./.github/workflows/build-container-image.yml
+    with:
+      file_to_build: streaming/Dockerfile
+      platforms: linux/amd64,linux/arm64
+      use_native_arm64_builder: true
+      cache: false
+      push_to_images: |
+        tootsuite/mastodon-streaming
```

Comment:
Should we still push it to Docker Hub? I remember the team mentioning that the `tootsuite` org on Docker Hub was legacy and deprecated, so we could only push to GHR

Reply:
Didn't they walk some of that back, or maybe it didn't apply to us like we thought?